Added some new features to WebUI

foxman69 opened the pull request:
Full change-list below:
1) Added Master schedule button
2) Added nicknames for each switch
3) Added an option to change username when logging in using WebUI
4) Fixed the time in scheduling to have a leading zero when having a 1 digit hour or minute

mcspr commented on code/espurna/scheduler.ino:
```diff
@@ -136,7 +137,11 @@ void _schCheck() {
     time_t utc_time = ntpLocal2UTC(local_time);
-
+    
+    if (getSetting("schAll").toInt() == 0) {
```
`schEnabled`?

foxman69 replied:
schEnabled is already taken (its the toggle of each schedule), i changed to to schMasterEnabled so its more indicative

mcspr commented on code/html/index.html:
```diff
@@ -1888,6 +1902,10 @@ <h2>
             </div>
+            <div class="p-g">
+                <div class="p-u-1 p-u-lg-1-4"><label>Nickname</label></div>
+                <div class="p-u-1 p-u-lg-1-4"><input name="relayNickname" class="p-u-1" type="text" original="" /></div>
```
original= is a dynamic attr, no need to set it

foxman69 replied:
deleted it

foxman69 commented:
I have fixed what you have mentioned (deleted the original attribute and changed the schAll name)
I actually see the "nickname" option as more indicative than label or tag, but you change choose what ever you like

I did changed the input type to input=time and deleted all the related js about leading zeros (when using input=time there is no need to this fix)

foxman69 commented:
I did saw that the input=time in android is working well looks a bit cut when the phone is on vertical mode, I will check this weekend how to fix it...

foxman69 commented:
I have fixed the problem in android and now it looks great
also made sure that there would always be a value when adding new schedule (Default value is 00:00)

mcspr commented:
Note that I have not forgotten about this, but I had a bit of a different idea for the relay naming.
If you followed the latest updates, one problem that we had was excessive payload size with repeating sensor descriptions whenever thingspeak or domoticz were enabled. Instead of sending those each time, sensor callback now sends those just once and caches them in the global 'magnitude' array.
Currently, we send out 'friendly' name via
https://github.com/xoseperez/espurna/blob/43a44cf96447dc2e04f0ef936573950df359889e/code/espurna/relay.ino#L693
That shows GPIO name, CH when it is light etc. What it can do instead is to check if `relayNicknameN` exists and send it instead of generating anything. SWITCHES config already shows that name, STATUS page needs a small adjustment to use it instead of generic 'Switch #'

foxman69 commented:
I will check the latest updates and adjust my code in the next few days
